docs/generate_extension_db.py

```diff
@@ -34,6 +34,8 @@ def IsMissing(value):
 
 
 def GetExtensionMetadata(target):
+  if not BUILDOZER_PATH:
+    raise ExtensionDbError('Buildozer not found!')
```

Comment:
im curious was this a fix for an issue you hit building the docs ?

Reply:
Exactly.. I hit some errors because the `buildozer` not found but the message was not clear about it and to discover it I needed to debug the python script. So I added this small exception do make our live easier.